CMake/FindQE.cmake

```diff
@@ -0,0 +1,26 @@
+# Locate QE via the patched pw2qmcpack.x
+# Require both to be present to consider QE_FOUND
+# Take QE_BIN as hint for location
+
+IF(QE_BIN)
+  FIND_PATH(QE_PW2Q_DIR pw2qmcpack.x ${QE_BIN})
+  FIND_PATH(QE_PW_DIR pw.x ${QE_BIN})
+ELSE(QE_BIN)
+  FIND_PATH(QE_PW2Q_DIR pw2qmcpack.x)
+  FIND_PATH(QE_PW_DIR pw.x)
+ENDIF(QE_BIN)
```

Comment:
No need of if(QE_BIN)
The following line allows QE_BIN being empty and if not it is preferred over system PATH.
```
FIND_PATH(QE_PW_DIR pw.x HINTS ${QE_BIN})
FIND_PATH(QE_PW2Q_DIR pw2qmcpack.x HINTS ${QE_BIN})
```